### PR TITLE
[FEATURE] added country taxes

### PR DESCRIPTION
**Pull request by turbo-ele**
Added countryTaxes

### CODE REVIEW

File: DataFixtures/ORM/CountryTaxes/LoadCountryTaxes.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        'reduced3' => 4,
+    );
+
+    private static $taxClassEntityName = 'SuluProductBundle:TaxClass';
+    private static $countryEntityName = 'SuluContactBundle:Country';
+
+    private $countries = array();
+    private $manager;
+
+//    private static $translations = ['de', 'en'];
```
Comment (turbo-ele) [inline DataFixtures/ORM/CountryTaxes/LoadCountryTaxes.php]: remove

File: DataFixtures/ORM/CountryTaxes/LoadCountryTaxes.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use Sulu\Bundle\ProductBundle\Entity\CountryTax;
+
+class LoadCountryTaxes implements FixtureInterface, OrderedFixtureInterface
+{
+
+    // Adapt these if there are any changes in database
+    private static $taxClassMappings = array(
```
Comment (christianbader) [inline DataFixtures/ORM/CountryTaxes/LoadCountryTaxes.php]: use []

File: Api/TaxClass.php [hunk 3]
```diff
@@ -75,4 +79,28 @@ public function getTranslation($locale)
...
+     * @return TaxClassTranslation
+     */
+    public function getCountryTaxes()
+    {
+        $taxes = null;
+        if (!$this->entity->getCountryTaxes()->isEmpty()) {
+            foreach ($this->entity->getCountryTaxes() as $countryTax) {
+                // If countryId is defined, show only tax of this country
+                if (!$this->countryId ||
+                    ($this->countryId && $countryTax->getCountry()->getId() == $countryTax)
```
Comment (turbo-ele) [inline Api/TaxClass.php]: please check if it shouldn't === $countryTax->getId()

File: Api/TaxClass.php [hunk 4]
```diff
@@ -75,4 +79,28 @@ public function getTranslation($locale)
 
         return $translation;
     }
-} 
+
+    /**
+     * @VirtualProperty
+     * @SerializedName("countryTaxes")
+     *
+     * @return TaxClassTranslation
```
Comment (turbo-ele) [inline Api/TaxClass.php]: hm?

File: DataFixtures/country-taxes.xml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    <country-tax>
+        <tax-class>standard</tax-class>
+        <country>AT</country>
+        <tax>20</tax>
+    </country-tax>
+    <country-tax>
+        <tax-class>reduced1</tax-class>
+        <country>AT</country>
+        <tax>10</tax>
+    </country-tax>
```
Comment (turbo-ele) [inline DataFixtures/country-taxes.xml]: shouldn't there be a reduced tax of 13% as well?

File: Entity/CountryTax.php [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+namespace Sulu\Bundle\ProductBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sulu\Bundle\ContactBundle\Entity\Country;
+use Sulu\Bundle\ProductBundle\Entity\TaxClass;
+
+class CountryTax
+{
+    /**
+     * @var integer
```
Comment (turbo-ele) [inline Entity/CountryTax.php]: int

**Comment by turbo-ele** [comment]
else ack

**Pull request merged**